src/Helpers/APIHelper.js

```diff
@@ -109,16 +109,17 @@ export function getCveListByAccount(apiProps, shouldUseHybridSystemFilter = fals
     return result;
 }
 
-export function getCveIdsList(apiProps) {
+export function getCveIdsList(apiProps, shouldUseHybridSystemFilter) {
```

Comment:
It seems your changes are missing. Can we do the same with `getCveListByAccount`?

Reply:
Sure
I have added those changes 👍🏼